dfastmi/gui/dialog_utils.py

```diff
@@ -247,3 +250,53 @@ def gui_text(
     except KeyError:
         application_setting = cstr[0]
     return application_setting
+
+
+def open_pdf_windows(pdf_path : str):
+    pdf_reader_path = get_default_pdf_reader_windows()
+    if pdf_reader_path:
+        try:
+            # Ensure the path is absolute
+            pdf_path = os.path.abspath(pdf_path)
+
+            # Open the PDF file with the default PDF reader in a non-blocking way
+            subprocess.Popen([pdf_reader_path, pdf_path], close_fds=True)
+        except (SystemExit, KeyboardInterrupt) as exception:
```

Comment:
is this double except needed if you use the same exception message twice? Or do you raise the error to a higher lvl?

Reply:
we discussed this and leave as is

Reply:
Discussed, this is done for a message in the qt view.